uint32 user_status
uint64 algorithm_status
float32 heading
float32 roll
float32 pitch
float32 north_speed
float32 east_speed
float32 vertical_speed
float64 latitude
float64 longitude
float32 altitude
uint32 timestamp
float32 heading_error_stddev
float32 roll_error_stddev
float32 pitch_error_stddev
float32 north_speed_error_stddev
float32 east_speed_error_stddev
float32 vertical_speed_error_stddev
float32 latitude_error_stddev
float32 longitude_error_stddev
float32 altitude_error_stddev
